db/schema.rb

```diff
@@ -303,8 +301,6 @@
   add_foreign_key "audits", "planning_applications"
   add_foreign_key "description_change_validation_requests", "planning_applications"
   add_foreign_key "description_change_validation_requests", "users"
-  add_foreign_key "notes", "planning_applications"
```

Comment:
you need to rebase and re-migrate as this is undoing Ben's work, looks like :smile: